### PR TITLE
added new ideas

### PR DESCRIPTION
**Pull request by Jwyman328**
Here is my new ideas

### CODE REVIEW

File: documentation.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a new idea
```
Comment (Jwyman328) [inline documentation.md]: I think this is a bad idea

Comment (jevidon) [reply documentation.md]: Why do you think this is a bad idea?

Comment (jevidon) [reply documentation.md]: @MGrogono can you confirm?

**Review by jevidon — APPROVED** [review]
lgtm

**Pull request merged**